src/tables.js

```diff
@@ -472,6 +472,17 @@ Flow.parseNoteData = noteData => {
     }
   } else {
     type = durationStringData.type;
+
+    // If we have keys, try and check if we've got a custom glyph
+    if (noteData.keys !== undefined) {
+      const regexp = /^([a-gx]?)\/(\d?)\/?([a-z][0-3])$/;
+      const result = regexp.exec(noteData.keys[0]);
+
+      // We have a custom glyph specified after the note eg. /X2
+      if (result && result.length === 4) {
+        type = result[3]; // Set the type to the custom note head
+      }
```

Comment:
Not sure I understand what's being attempted here. (Also, the regex stuff up there is probably going to be fragile, is there a better way to do this?)

Reply:
The type for the note defaults to `n`.  When, for example, `keys: ['a/5/x3']` is specified (circle-x note head) then I set the type to `x3`, the custom note head identifier.  The reason for this is because the width of the head is calculated wrong for custom note heads.
An alternative would be to implement a new `Flow.durationToGlyph.duration_codes` for each custom head, but that seems a lot of duplication.

The regex can be replaced by `str.split('/');` but I am not sure whether about performance impact and believed the regex to be faster.

Reply:
Yes please, `str.split('/')` is much clearer about intent and far less fragile.